public/vendor/tegaki/0.9.4/tegaki.css

```
@@ -0,0 +1,669 @@
+/*! Contains fonts from Font Awesome (Copyright (C) 2016 by Dave Gandy), Entypo (Copyright (C) 2012 by Daniel Bruce) */
+@font-face {
+  font-family: 'tegaki';
```

Stylelint (17.12.0)

[error] Expected no quotes around "tegaki" (font-family-name-quotes)

(font-family-name-quotes)